### PR TITLE
News site project

### PR DESCRIPTION
**Pull request by Aepera**

### CODE REVIEW

File: code/index.html [hunk 1]
```diff
@@ -1,22 +1,144 @@
...
-    <section class="">
-      <!-- The other news  -->
-    </section>
-  </body>
-</html>
+
+<head>
+  <meta charset="utf-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Project 2: The News Site</title>
```
Comment (HIPPIEKICK) [inline code/index.html:7]: You can change this to something personal!

File: code/index.html [hunk 2]
```diff
@@ -1,22 +1,144 @@
...
+      <i class="footer-logo fab fa-twitter"></i>
+      <i class="footer-logo fab fa-facebook-f"></i>
+      <i class="footer-logo fab fa-instagram"></i>
+      <i class="footer-logo fas fa-envelope"></i>
+    </div>
+  </footer>
+
+</body>
+
+</html>
```
Comment (HIPPIEKICK) [inline code/index.html:144]: Really good structure of this HTML file, great job! 💯

File: code/index.html [hunk 3]
```diff
@@ -1,22 +1,144 @@
...
+        <div></div>
+        <div></div>
+      </div>
+    </a>
+  </header>
+
+  <!-- BIG NEWS SECTION -->
+  <section class="big-news container">
+    <div class="big-news-img">
+      <img src="./images/river-5662293_640.jpg" alt="River image">
```
Comment (HIPPIEKICK) [inline code/index.html:41]: I'm longing for mountains right now!

File: code/style.css [hunk 4]
```diff
@@ -1,4 +1,191 @@
...
+  color: #14274e;
+}
+
+.small-news img {
+  width: 100%;
+}
+
+.small-news img:hover {
+  box-shadow: 0px 5px 0px 5px #394867;
+}
```
Comment (HIPPIEKICK) [inline code/style.css:110]: This box-shadow kind of overlaps on the sibling images. Maybe you can allocate some space in between the elements or just have the box shadow at the bottom

File: README.md [hunk 5]
```diff
@@ -1,12 +1,14 @@
...
-Replace this readme with your own information about your project. 
-
-Start by briefly describing the assignment in a sentence or two. Keep it short and to the point.
+The aim of this project was to build a news/magazine website, using Flexbox technique to create the structure and responsiveness to viewport sizes.
 
 ## The problem
 
-Describe how you approached to problem, and what tools and techniques you used to solve it. How did you plan? What technologies did you use? If you had more time, what would be next?
+I used Flexbox to build the structure and layout of the website. 
+Since the site should be responsive, I started with coding for mobile view and then added media queries for tablet and desktop sizes.
```
Comment (HIPPIEKICK) [inline README.md:8]: 👍

**Review by HIPPIEKICK — APPROVED** [review]
Nice job Astrid! Your design is really nice and aligned, the structure of your HTML and CSS is neat and you managed well with the responsiveness 💯 Optional challenge to you: Think about what you can do to make your website more dynamic, e.g. add `cursor: pointer;` to clickable elements, adding more hover effects or animations.

We’ve also put together some general feedback that applies to many projects this first week. Check it out and take the tips with you to upcoming projects! 👉 [General Feedback - week 2](https://www.notion.so/technigo/Week-2-Project-news-site-bbe695289d154ff98d6a247cef873f07)

File: code/index.html (hunk 1 above)
Comment (Aepera) [reply code/index.html:7]: Thank you for your nice feedback! 😊 
Should I click on Close pull request, if I consider myself to be "done" with this project for now?
Or can it stay open if I want to come back to it later this spring?

**Pull request closed without merging**